app/core/middleware.py

```
@@ -105,12 +107,22 @@ async def dispatch(self, request: Request, call_next) -> Response:
         request.state.real_ip = real_ip
-
-        response = await call_next(request)
-
-        # 로깅 (정상 헬스체크는 스킵하되, 500번대 에러 발생 시 상태 코드 포함시켜 로깅)
-        if request.url.path not in ("/ping", "/health") or (response is not None and response.status_code >= 500):
-            status_code = response.status_code if response else 500
+        
+        start_time = time.perf_counter()
+        try:
+            response = await call_next(request)
+        except Exception as e:
```

Ruff (0.15.2)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)